src/modules/BootStorage.rb

```diff
@@ -176,7 +176,8 @@ def bootloader_installable?
     def detect_disks
       # The AutoYaST config mode does access to the system.
       # bnc#942360
-      return :ok if Mode.config
+      return if Mode.config
```

Comment:
well, for me it is better to not merge non-related stuff together. First return is if Mode Config is used and needs to be skipped. The second return is caching.
I think it make sense to put thinks in one condition if it is related like `return if !@RootPartitionDevice && !@RootPartitionDevice.empty?` or `return if !@BootPartitionDevice.empty? || !@RootPartitionDevice.empty?` as both is related to caching.